data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/s3/SqsWorker.java

```diff
@@ -263,6 +265,9 @@ && isEventBridgeEventTypeCreated(parsedMessage)) {
                             parsedMessageVisibilityTimesMap.remove(parsedMessage);
                         }
                         if (result == true) {
+                            try {
+                                Thread.sleep(40*1000);
```

Comment:
Why 40 seconds? Is it related to another constant?

It would be good to make this a constant with a useful name.